<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lb_nome">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>NOME</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_cpf">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>80</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_email">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EMAIL</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_telefone">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TELEFONE</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_responsavel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>RESPONSAVEL</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_procedimento">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>200</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>PROCEDIMENTO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botao_inserir">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>300</y>
      <width>141</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>INSERIR</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_5">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="campo_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
